apps/dashboard/pkg/migration/migrate_test.go

```diff
@@ -99,33 +87,30 @@ func testMigration(t *testing.T, dash map[string]interface{}, name string, input
 	require.JSONEq(t, string(existingBytes), string(outBytes), "%s did not match", outPath)
 }
 
-func parseInputName(t *testing.T, name string) (int, string) {
+func getSchemaVersion(t *testing.T, dash map[string]interface{}) int {
 	t.Helper()
-	parts := strings.SplitN(filepath.Base(name), ".", 3)
-	if len(parts) < 3 {
-		t.Fatalf("invalid input filename: %s", name)
+	version, ok := dash["schemaVersion"]
+	require.True(t, ok, "dashboard missing schemaVersion")
```

Comment:
FYI eventually this is going to fail  as we are going to have a dashboard without a schema version, which is fairly common scenario from the data I think Thanos pulled at some point. In such case we should treat such a dashboard as schemaVersion = 0.

Reply:
Ok! Yep, I understand. This is only for output generation. The runtime functions and has been tested. I agree this will fail when we add an input without schemaVersion, but for now we shouldn't do that because it will fail the output comparison every time.

Reply:
Not really - i think leaving this as-is is ok, when we add no schemaVersion test this will fail and we can adjust it then.